src/System.Collections.Immutable/dir.props

```diff
@@ -5,5 +5,6 @@
     <PackageVersion>1.4.0</PackageVersion>
     <AssemblyVersion>1.2.2</AssemblyVersion>
     <IsNETCoreApp>true</IsNETCoreApp>
+    <IsUAP>true</IsUAP>
```

Comment:
IIRC it was because they were in the previous stable package, so we needed to have it in there for the new one as well. This was the same case for the WCF assemblies.